experimental/GModule/src/GModule.jl

```diff
@@ -807,7 +807,7 @@ gmodule(k::fpField, C::GModule{<:Any, <:AbstractAlgebra.FPModule{fpFieldElem}})
       continue
     end
     T = action(C, r)
-    push!(chr, (c, trace(matrix(T))))
+    push!(chr, (c, K(trace(matrix(T)))))
```

Comment:
Out of curiosity, how could that not be in `K`?

Reply:
Not sure.. I am scared about the default field(s), the GAP <-> Oscar FF conversion and algebraic closures...